sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -157,6 +157,9 @@ case class DropFunction(
         throw new AnalysisException(s"Specifying a database in DROP TEMPORARY FUNCTION " +
           s"is not allowed: '${databaseName.get}'")
       }
+      if (FunctionRegistry.builtin.functionExists(functionName)) {
```

Comment:
Overwriting a built-in function is missing in the current implementation of the current `FunctionRegistry`. Should we support it in this release?